services/minespace-web/src/components/Forms/noticeOfDeparture/EditNoticeOfDepartureForm.js

```diff
@@ -25,13 +25,14 @@ const propTypes = {
   onSubmit: PropTypes.func.isRequired,
   closeModal: PropTypes.func.isRequired,
   handleSubmit: PropTypes.func.isRequired,
+  pristine: PropTypes.bool.isRequired,
   mineGuid: PropTypes.string.isRequired,
   noticeOfDeparture: CustomPropTypes.noticeOfDeparture.isRequired,
 };
 
 // eslint-disable-next-line import/no-mutable-exports
 let AddNoticeOfDepartureForm = (props) => {
```

Comment:
Just noticed this should be been
```suggestion
let EditNoticeOfDepartureForm = (props) => {
```
right?

Reply:
:man_facepalming: Indeed.  All of the references on this page are for `AddNoticeOfDepartureForm`.  I'll update them all and test quickly.

Reply:
Good catch!

Reply:
Okay, this has been updated now.